whisper/stt/processing/utils.py

```diff
@@ -65,11 +68,8 @@ def get_language(language = None):
         language = {v: k for k, v in LANGUAGES.items()}.get(language.lower(), language)
```

Comment:
you see the lower here.
I think the one at line 62 is not needed.

Reply:
What is the purpose of that line, not sure to understand what it does?

Reply:
I just mention that the lower was done somewhere else.
Read again my comment : look for "lower()" in the code, it appeas already where needed

Also, I think "French" (capitalized) is among the official list of languages by Whisper.
And we both agree that it is about have case insensitive input for the user.

Reply:
> you see the lower here. I think the one at line 62 is not needed.

I was just saying that the one that is not needed is the one at line 68 not the one at line 62. If we keep the one at line 68 and not the one at line 62 we can't use "FR" or "Fr" as language argument